app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,57 @@
+class PasswordResetsController < ApplicationController
+  before_action :get_user, only: [:edit, :update]
+  before_action :valid_user, only: [:edit, :update]
+  before_action :check_expiration, only: [:edit, :update]
+
+  def new; end
+
+  def edit; end
+
+  def create
+    @user = User.find_by email: params[:password_reset][:email].downcase
+    if @user
+      @user.create_reset_digest
+      @user.send_password_reset_email
+      flash[:info] = "Email sent with password reset instructions"
+      redirect_to root_url
+    else
+      flash.now[:danger] = "Email address not found"
+      render :new
+    end
+  end
+
+  def update
+    if params[:user][:password].empty?
```

Comment:
```suggestion
    if params[:user][:password].blank?
```
phân biệt giúp anh blank? empty? nil?